### PR TITLE
feat: tooltip theme

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

#### tooltip

```ts
type FontTheme = {
  fontSize?: number;
  fontFamily?: string;
  fontWeight?: string | number;
  color?: string;
};

type TooltipTheme = {
  background?: string;
  borderColor?: string;
  borderWidth?: number;
  borderRadius?: number;

  header?: FontTheme;
  body?: FontTheme;
};
```

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by shiren** [comment]
LGTM

**Review by lja1018 — APPROVED** [review]
[11/10] 리뷰완료
고생하셨습니다!

File: src/helpers/tooltipTemplate.ts [hunk 1]
```diff
@@ -21,16 +28,25 @@ function getTitleValueTemplate(title: string, value: string) {
...
 
-export function getDefaultTemplate(model: TooltipModel, { header, body }: DefaultTooltipTemplate) {
-  return `<div class="tooltip">${header}${body}</div>`;
+export function getDefaultTemplate(
+  model: TooltipModel,
+  { header, body }: DefaultTooltipTemplate,
+  theme: Required<TooltipTheme>
+) {
+  const { borderColor, borderWidth, background, borderRadius } = theme;
+  const style = `border: ${borderWidth}px solid ${borderColor};border-radius: ${borderRadius}px;background: ${background};`;
```
Comment (jungeun-cho) [inline src/helpers/tooltipTemplate.ts:37]: borderStyle도 열어주는거 어때요?

File: types/theme.d.ts [hunk 2]
```diff
@@ -52,6 +53,16 @@ type AxisTheme = {
   color?: string;
 };
 
+type TooltipTheme = {
+  background?: string;
+  borderColor?: string;
+  borderWidth?: number;
+  borderRadius?: number;
+
```
Comment (jungeun-cho) [inline types/theme.d.ts:61]: 빈줄 삭제요~

**Review by jungeun-cho — APPROVED** [review]
[11/10] 이견없습니다.ㅎ 수고하셨어요~

File: src/helpers/tooltipTemplate.ts (hunk 1 above)
Comment (jung-han) [reply src/helpers/tooltipTemplate.ts]: 반영해씁니다~

**Pull request merged**